cps/src/test/java/org/jenkinsci/plugins/workflow/cps/AbstractCpsFlowTest.java

```diff
@@ -50,7 +50,14 @@
     public TemporaryFolder tmp = new TemporaryFolder();
 
     @Rule
-    public JenkinsRule jenkins = new JenkinsRule();
+    public JenkinsRule jenkins = new JenkinsRule() {
+        @Override public void before() throws Throwable {
+            if (Thread.interrupted()) {
+                System.err.println("was interrupted before start");
+            }
+            super.before();
```

Comment:
Yes, we still want to run the test, we merely want to clear (ignore) the interrupt flag.